src/main.js

```diff
@@ -473,6 +494,14 @@ define(["storage", "websockets", "account/morph-props"], function(storage, ws, p
         return audioLabel;
     }
 
+    function appendAudio(elementID, src) {
+        var audioElement = document.createElement("audio");
```

Comment:
Can we do this with `var audio = new Audio('audio_file.mp3'); audio.play();`?

Reply:
updated